feature/mainflow/flow/build.gradle.kts

```diff
@@ -4,26 +4,22 @@ plugins {
 }
 
 android {
-    namespace = "com.epmedu.animeal.tabs"
+    namespace = "com.epmedu.animeal"
```

Comment:
something strange with package now)

Reply:
What do you mean? Do you want me to change it to `com.epmedu.animeal.main`?